### PR TITLE
Challenge

### PR DESCRIPTION
**Pull request by imlisbeth**

### CODE REVIEW

File: Viewer/Viewer/DetailViewController.swift [hunk 1]
```diff
@@ -10,12 +10,14 @@ import UIKit
...
     var selectedImage: String?
-    
+    var selectedPictureNumber = 0
+    var totalPictures = 0
     
     override func viewDidLoad() {
         super.viewDidLoad()
         
-        title = selectedImage
+//      title = selectedImage
```
Comment (rnaben) [inline Viewer/Viewer/DetailViewController.swift:19]: Remove this

File: Viewer/Viewer/ViewController.swift [hunk 2]
```diff
@@ -27,7 +27,11 @@ class ViewController: UITableViewController {
                 pictures.append(item)
             }
         }
-        print(pictures)
+//      pictures = pictures.sorted(by: { item1, item2 in
```
Comment (rnaben) [inline Viewer/Viewer/ViewController.swift:30]: Remove this and the two subsequent lines

File: Viewer/Viewer/ViewController.swift [hunk 3]
```diff
@@ -27,7 +27,11 @@ class ViewController: UITableViewController {
...
+//          item1 < item2
+//      })
+        pictures = pictures.sorted(by: { $0 < $1 } )
+//      print(pictures)
```
Comment (rnaben) [inline Viewer/Viewer/ViewController.swift:34]: Remove this

File: Viewer/Viewer/ViewController.swift [hunk 4]
```diff
@@ -45,8 +49,11 @@ class ViewController: UITableViewController {
         if let vc = storyboard?.instantiateViewController(withIdentifier: "Detail") as? DetailViewController {
             // 2: success! Set its selectedImage property
             vc.selectedImage = pictures[indexPath.row]
+            // 3.
```
Comment (rnaben) [inline Viewer/Viewer/ViewController.swift:52]: Complete or remove this

**Review by rnaben — CHANGES_REQUESTED** [review]
Do it!

File: Viewer/Viewer/ViewController.swift (hunk 4 above)
Comment (imlisbeth) [reply Viewer/Viewer/ViewController.swift:52]: Done!

File: Viewer/Viewer/ViewController.swift (hunk 3 above)
Comment (imlisbeth) [reply Viewer/Viewer/ViewController.swift:34]: Done!

File: Viewer/Viewer/DetailViewController.swift (hunk 1 above)
Comment (imlisbeth) [reply Viewer/Viewer/DetailViewController.swift:19]: Done!

File: Viewer/Viewer/ViewController.swift (hunk 2 above)
Comment (imlisbeth) [reply Viewer/Viewer/ViewController.swift]: Done!

**Review by rnaben — APPROVED** [review]

**Pull request merged**